src/dispatcher.js

```diff
@@ -292,10 +292,25 @@ var dispatcher = {
     return eventCopy;
   },
   getTarget: function(inEvent) {
-
-    // if pointer capture is set, route all events for the specified pointerId
-    // to the capture target
-    return this.captureInfo[inEvent.pointerId] || inEvent._target;
+    var capture = this.captureInfo[inEvent.pointerId];
+    if (!capture) {
+      return inEvent._target;
+    }
+    if (!this.suppressDuringCapture(inEvent)) {
+      return capture;
+    }
+  },
+  suppressDuringCapture: function(inEvent) {
+    var suppress = [
```

Comment:
Move this to a constant (like `var CLONE_DEFAULTS` above) and make it an object to improve lookup performance. Use "BOUNDARY_EVENTS" for the name.